src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingTxWithDoubleSpendSpecification_Steps.cs

```diff
@@ -100,8 +100,7 @@ private void receiving_node_attempts_to_double_spend_mempool_doesnotaccept()
 
         private void trx_is_mined_into_a_block_and_removed_from_mempools()
         {
-            new SharedSteps().MineBlocks(1, this.stratisSender, "account 0", "mywallet", "123456", 16360L);
-
+            new SharedSteps().MineBlocks(1, this.stratisSender, "account 0", "mywallet", "123456");
```

Comment:
there is no more fees for the mine block?

Reply:
The test fails if I do not change this.